optimade/server/entry_collections/entry_collections.py

```diff
@@ -82,12 +82,11 @@ def find(
 
     @property
     def all_fields(self) -> set:
-        """ Get the set of all fields handled in this collection, from
-        attribute fields in the schema, provider fields and top-level
-        OPTIMADE fields.
+        """Get the set of all fields handled in this collection,
+        from attribute fields in the schema, provider fields and top-level OPTIMADE fields.
 
-    Returns:
-        set: All fields handled in this collection.
+        Returns:
+            All fields handled in this collection.
```

Comment:
```suggestion
           All fields handled in this collection.
```

Reply:
I think this is all right, no? It's a 4 space indentation already, right?

Reply:
The highlighting in the GitHub editor made it look off-by-one to me, but on closer inspection you're right